openai/api_resources/image.py

```diff
@@ -10,8 +10,11 @@ class Image(APIResource):
     OBJECT_NAME = "images"
 
     @classmethod
-    def _get_url(cls, action):
-        return cls.class_url() + f"/{action}"
+    def _get_url(cls, openai_action, azure_action, api_type, api_version):
```

Comment:
I would suggest not renaming the parameter here. It would also be great to reduce the number of parameters that need to be kept in sync are there any scenarios where you would specify `azure_action` but not have the `api_type` be one of the Azure ones?

Reply:
I reversed the rename. In terms of the parameters, we probably don't need to check for `azure_action` but from the way we are calling it, we are always passing the `azure_action` and if we use it or not depends on the `api_type`.
This made me think that we probably want to handle calls to the other APIs (`edits`, and `variations`). Do you think we should raise an exception there if they are called with azure `api_type`s?